src/test/mission.test.js

```diff
@@ -0,0 +1,68 @@
+import {
+  render, screen, act, fireEvent,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Missions from '../pages/Missions';
+
+describe('Missions page component', () => {
+  afterEach(() => {
+    act(() => store.dispatch({
+      type: 'spacetravelhub/missions/fetch',
+      payload: [],
+    }));
+  });
```

Comment:
- [ ] One of your tests is failing
![Screenshot from 2022-08-26 10-55-11](https://user-images.githubusercontent.com/65860678/186869310-b2c27ac2-2b20-456a-9ddc-a31b5b0c1817.png)

Reply:
Dear @uwadonat, actually I don't know why its failed from you but everything is correct in my computer. this is a screenshot of my run test >
![test-spaceX](https://user-images.githubusercontent.com/35001742/186875866-afdb552b-07f6-4b53-bd72-397330b91af4.JPG)